source/dofs/dof_handler_policy.cc

```diff
@@ -4064,7 +4041,7 @@ namespace internal
               {
                 std::vector<dealii::types::global_dof_index> local_dof_indices;
 
-                typename DoFHandlerType::level_cell_iterator
+                typename DoFHandler<dim, spacedim>::level_cell_iterator
                   cell = dof_handler->begin(level),
                   endc = dof_handler->end(level);
```

Comment:
I the second I pressed `approve` I saw these objects. Can we convert the place here and a few more below to range-based for loops as well?

Reply:
Done!